kodilla-advanced-tests/src/test/java/com/kodilla/parametrized_tests/homework/GamblingMachineTestSuite.java

```diff
@@ -0,0 +1,67 @@
+//package com.kodilla.parametrized_tests.homework;
+//
+//import org.junit.jupiter.api.Assertions;
+//import org.junit.jupiter.api.Test;
+//import org.junit.jupiter.params.ParameterizedTest;
+//import org.junit.jupiter.params.provider.CsvFileSource;
+//
+//import java.util.Collections;
+//import java.util.HashSet;
+//import java.util.Set;
+//
+//public class GamblingMachineTestSuite {
+//
+//    @ParameterizedTest
+//    @CsvFileSource(resources = "/GoodNumbers.csv")
+//    public void shouldCheckWinsWorks(int value1, int value2, int value3, int value4, int value5, int value6)
```

Comment:
public void shouldCheckWinsWork(String numbers) {
    String[] parsedNumbers = numbers.split(",");
   Set<Integer> numbersInt = new HashSet<>();
    for (String number : parsedNumbers) {
      numbersInt.add(Integer.valueOf(number));
   }
}